ts/components/wallet/card/CardComponent.tsx

```diff
@@ -207,23 +209,16 @@ export default class CardComponent extends React.Component<Props> {
     };
 
     const expirationDate = buildExpirationDate(creditCard);
-    const isExpired = isExpiredCard(creditCard);
+    const isCardExpired = isExpired(
+      creditCard.expireMonth,
+      creditCard.expireYear
+    ).getOrElse(false);
```

Comment:
What you think about create a method `isPaymentMethodExpired = (pm: PaymentMethod): Option<boolean>` like we did in https://github.com/pagopa/io-app/blob/d8cd34c0e3106b517b7da0fd998666fa760b4a7f/ts/utils/paymentMethod.ts#L43 ? 
In this way if we need the information elsewhere in the code, we can avoid to duplicate this piece of code and we can have this information (is expired) associated in a generic way to a `PaymentMethod`

Reply:
It sounds good!
Within these commits https://github.com/pagopa/io-app/pull/2982/commits/689fdc4eff4edf999b35169a144a7008bca4949b https://github.com/pagopa/io-app/pull/2982/commits/a57becfe7dee807500e27d97ca26edd4c21f4d1a

- changed the return type of `isExpired` from `Option` to `Either`
- created `isPaymentMethodExpired` function as shortcut for `RawPaymentMethod`
- add a property for `Bancomat` and `CoBadge` card component to receive if that card is expired or not
- update relative tests